dev-packages/application-package/src/extension-package.ts

```diff
@@ -25,6 +25,10 @@ export interface Extension {
     backend?: string;
     backendElectron?: string;
     electronMain?: string;
+    /**
+     * Map of script name to script relative path
```

Comment:
We need doc describing what kind of value should be put here. What does "script-relative-path" mean? Relative to the package.json? Should it be './foobar.js', do we reference the *.ts file or do we reference the compiled version?

Reply:
Will change to

> Map of entry point id to js script path relative to its package root.